scadnano/scadnano.py

```diff
@@ -4133,23 +4133,19 @@ def _cadnano_v2_place_strand(self, strand: Strand, dct: dict,
                 self._cadnano_v2_place_crossover(which_helix, next_helix,
                                                  domain, next_domain, strand_type)
 
-    def _cadnano_v2_fill_blank(self, dct: dict, num_bases: int) -> Dict[int, int]:
+    def _cadnano_v2_fill_blank(self, dct: dict, num_bases: int, design_grid: Grid) -> dict:

```

Comment:
Did I get the type wrong here? I had annotated the return type of `_cadnano_v2_fill_blank` as `Dict[int, int]` based on looking at usages in the definition.

If that's wrong, can we specify a more specific type than `dict`? Or do the types of the the values and keys vary? 

If only the values vary, but the keys are always `int`, then it's still better to use `Dict[int, Any]` (instead of `dict`)